src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionService.java

```diff
@@ -286,6 +288,19 @@ private void deleteFromS3(String requestId, String attachmentId, Map<String, Obj
         }
     }
 
+    public FileTransferApiClientResponse downloadAttachment(String requestId,
+                                                            String objectionId,
+                                                            String attachmentId,
+                                                            HttpServletResponse response) throws ServiceException {
+        Map<String, Object> logMap = buildLogMap(null, objectionId, attachmentId);
+        try {
+            return fileTransferApiClient.download(attachmentId, response);
+        } catch (IOException e) {
```

Comment:
Need a unit test for IOException being thrown

Reply:
We shold bear in mind that this is really only part of the stop gap and is likely to change.

Reply:
Added a test